cmd/eval-dev-quality/cmd/evaluate_test.go

```diff
@@ -1240,6 +1250,48 @@ func TestEvaluateInitialize(t *testing.T) {
 			}
 		},
 	})
+	validate(t, &testCase{
+		Name: "Load configuration",
+
+		Before: func(t *testing.T, workingDirectory string) {
+			configurationContent := bytesutil.StringTrimIndentations(`
+				{
+					"Models": {
+						"Selected": [
+							"symflower/symbolic-execution",
+							"openrouter/claude-3-sonnet"
+						]
+					},
+					"Repositories": {
+						"Selected": [
+							"golang/plain",
+							"golang/transpile",
+							"java/plain",
+							"java/transpile"
+						]
+					}
+				}
+			`)
+			require.NoError(t, os.WriteFile(filepath.Join(workingDirectory, "config.json"), []byte(configurationContent), 0700))
+		},
+
+		Command: makeValidCommand(func(command *Evaluate) {
+			command.Configuration = "config.json"
+		}),
+
+		ValidateCommand: func(t *testing.T, command *Evaluate) {
```

Comment:
Validate the output configuration itself as well please!

Reply:
I.e. in the output config we should again have the models and repositories that we had as inputs.